src/_pytest/runner.py

```diff
@@ -52,10 +52,19 @@ def pytest_addoption(parser: Parser) -> None:
         metavar="N",
         help="show N slowest setup/test durations (N=0 for all).",
     )
+    group.addoption(
+        "--durations-min",
+        action="store",
+        type=float,
+        default=0.005,
+        metavar="N",
+        help="Minimal duration in seconds for inclusion in slowest list. Default 0.005",
```

Comment:
```suggestion
        help="Minimal duration in seconds for inclusion in slowest list.",
```
Default is probably already appended by argparse, right?

Reply:
It is not, that is why I added it explicitly. I am not sure why but I know is not printed.